3.10/arangosync-migration-tool.md

```diff
@@ -0,0 +1,232 @@
+[how to create an API key](oasis/api-getting-started.html#creating-an-api-key).
+
+### Setting up the target deployment in Oasis
+
+Continue by [creating a new Oasis deployment](oasis/deployments.html#how-to-create-a-new-deployment)
+or choose an existing one.
+
+The target deployment in Oasis requires specific configuration rules to be
+set up before the migration can start:
+
+- **Configuration settings**: The target deployment must be compatible with the
+source data cluster. This includes the ArangoDB version that is being used,
+the DB-Servers count, and disk space.
+- **Deployment region and cloud provider**: Choose the closest region to your
+data cluster. This factor can speed up your migration to the cloud.
+
+After setting up your Oasis deployment, wait for a few minutes to become 
+fully operational.
+
+## Running the migration tool
+
+The `arangosync-migration` tool provides a set of commands that allow you to:
+-  start the migration process
+-  check whether your source and target clusters are fully compatible
+-  get the current status of the migration process
+-  stop or abort the migration process
+-  switch the local cluster to read-only mode
+
+### Starting the migration process
+
+To start the migration process, run the following command:
+
+```bash
+arangosync-migration start
+```
+
+An optional `--check-only` option can be provided. When specified, you can check
+if your local cluster and target deployment are compatible and if the migration 
+can start. No data is sent to Oasis.
+
+Before the actual migration starts, the `start` command also runs some
+pre-checks. Among other things, it measures the disk space which is occupied
+by your ArangoDB cluster. If you are using the same data volume for ArangoDB
+servers and other data as well, the measurements can be incorrect.
+Provide the `--source.ignore-metrics` option to overcome this.
+
+Once the migration starts, the local cluster enters into monitoring mode and the
+synchronization status is displayed in real-time. If you don't want to see the
+status you can also terminate this process, as the underlying agent process
+continues to work. If something goes wrong, restarting the same command restores
+the replication state.
+
+To restart the migration, use the `stop --abort` command.
+Note that restarting works only if you are not using
+auto-generated certificates.
+
+{% hint 'warning' %}
+Starting the migration creates a full copy of all data from the source cluster
+to the target deployment in Oasis. All data that has previously existed in the
+target deployment will be lost.
+{% endhint %}
+
+### During the migration...
+
+What happens during active migration:
+- The source data cluster remains usable. 
+- The target deployment in Oasis is switched to read-only mode.
+- Your root user password is not copied to the target deployment in Oasis.
+To get your new root password, select the target deployment from the Oasis
```

Comment:
It is not "new" root user password. It stays the same as it was before running migration.